src/app/core/rendering/shapes/ArcCircle.ts

```diff
@@ -2,16 +2,30 @@ import {Vector} from "Vector";
 
 import {Circle} from "./Circle";
 
+/**
+ * A representation of an ArcCircle shape.
+ */
 export class ArcCircle extends Circle {
     protected a0: number = 0;
     protected a1: number = 2*Math.PI;
 
+    /**
+     * Constructor for ArcCircle
+     * @param pos the position
```

Comment:
nit: Leave an empty line between the end of the description and the start of the tags like so:
```
 * Constructor for ArcCircle
 *
 * @param pos the position
```